cmd/diff/diff.go

```diff
@@ -0,0 +1,337 @@
+package main
```

Comment:
let's move this to `cmd/notary/diff.go` for consistency, and that way we'll pick up all of the duplicated helper functions as well.